hipify_torch/cuda_to_hip_mappings.py

```diff
@@ -7889,6 +7891,7 @@
             ("hipsparseDestroyMatDescr", CONV_MATH_FUNC, API_SPECIAL),
         ),
         ("cusparseDestroy", ("hipsparseDestroy", CONV_MATH_FUNC, API_SPECIAL)),
+        ("cusparseGetVersion", ("hipsparseGetVersion", CONV_MATH_FUNC, API_SPECIAL)),
         ("cusparseXcoo2csr", ("hipsparseXcoo2csr", CONV_MATH_FUNC, API_SPECIAL)),
         ("cusparseMatDescr_t", ("hipsparseMatDescr_t", CONV_MATH_FUNC, API_SPECIAL)),
```

Comment:
Please check into the difference between CONV_MATH_FUNC and CONV_TYPE it seems like some _t API's are labeled as  CONV_MATH_FUNC.

Reply:
Done.